main/utils/encoding.js

```diff
@@ -14,7 +14,11 @@ const getEncoding = async filePath => {
   try {
     await execa(ffmpegPath, ['-i', filePath]);
   } catch (error) {
-    return /.*: Video: (.*?) \(.*/.exec(error.stderr)[1];
+    const errorText = /.*: Video: (.*?) \(.*/.exec(error.stderr);
+    if (!errorText) {
+      throw error;
+    }
+    return errorText[1];
```

Comment:
@andreidobrinski, almost there. Looking at the [CI's error output](https://app.circleci.com/pipelines/github/wulkano/Kap/169/workflows/b44a6fd4-d110-464b-88e6-4f0952cddc63/jobs/1722), I believe the linter wants:
```javascript
    const errorText = /.*: Video: (.*?) \(.*/.exec(error.stderr);
    if (!errorText) {
      throw error;
    }

    return errorText[1];
```

We can use `yarn test` locally to verify.

Reply:
Thanks for the tip! I committed the update :)